src/test/java/com/puppycrawl/tools/checkstyle/xpath/RootNodeTest.java

```diff
@@ -386,19 +357,17 @@ public void testCopy() {
             fail("Exception is excepted");
         }
         catch (UnsupportedOperationException ex) {
-            assertEquals(
-                "Invalid exception message",
-                "Operation is not supported",
-                ex.getMessage());
+            assertEquals("Operation is not supported",
+                ex.getMessage(), "Invalid exception message");
```

Comment:
This two lines are duplicated many times. May be extract it to some method?

Reply:
We can think about this, but previously we tried to keep assert in test methods, no nested methods with asserts. It is not always true for our tests, but try to keep it. Minor code duplication is cheaper than dependency between tests (there is no certain line to clear define when to duplicate and when to make common method).